docs/reference/targets/smtp.md

```diff
@@ -65,14 +65,14 @@ Refuse to pass messages over plain-text connections.
 
 ---
 
-### auth `off` | `plain` _username_ _password_ | `forward`  | `external`
+### auth `off` | `plain|auth` _username_ _password_ | `forward`  | `external`
 Default: `off`
 
 Specify the way to authenticate to the remote server.
 Valid values:
 
 - `off` – No authentication.
-- `plain` – Authenticate using specified username-password pair.
+- `plain` or `auth` – Authenticate using specified username-password pair.
```

Comment:
"login"

Mentioning why there are two options would be useful too.
Something among the lines:

> "login" uses obsolete SASL LOGIN mechanism instead of SASL PLAIN. This is required for compatibility with some SMTP services (e.g. Office 365).

Reply:
Done :)